packages/core/src/command/hl7v2-subscriptions/hl7v2-to-fhir-conversion/shared.ts

```diff
@@ -41,6 +50,12 @@ export function unpackPidFieldOrFail(pid: string) {
   return { cxId, patientId };
 }
 
+export function getPatientIdsOrFail(msg: Hl7Message): { cxId: string; patientId: string } {
+  const pid = getSegmentByNameOrFail(msg, "PID");
+  const idComponent = pid.getComponent(3, 1);
+  return unpackPidFieldOrFail(idComponent);
+}
+
```

Comment:
hey i know you're just moving this and it was approved before, but i think this is actually misleading - we're unpacking the Pid field and getting a cx_id and patient id pair. Can we call it getCxIdAndPatientId or something?